tasks/main.yml

```diff
@@ -25,10 +25,19 @@
     update_cache: '{{ __systemd_timesyncd_virtualized }}'
   become: yes
 
+- name: Create drop-in configuration folder
+  ansible.builtin.file:
+    path: /etc/systemd/timesyncd.conf.d
+    state: directory
+    owner: root
+    group: root
+    mode: 0755
+  become: yes
+
 - name: configure systemd-timesyncd
   ansible.builtin.template:
-    src: timesyncd.conf.j2
-    dest: /etc/systemd/timesyncd.conf
+    src: local.conf.j2
+    dest: /etc/systemd/timesyncd.conf.d/local.conf
```

Comment:
I'd rather make it explicit that this comes from Ansible, please use `/etc/systemd/timesyncd.conf.d/ansible.conf`.

Reply:
Note that you also need to change `molecule/default/verify.yml`, it still points to `/etc/systemd/timesyncd.conf`.

Reply:
`README.md` needs a change as well.